### PR TITLE
Update breaking change docs

### PR DESCRIPTION
**Pull request by conniey**
# Update breaking change documentation from github.com/microsoft/dotnet repository

## Summary

Runs the tool against the breaking changes in https://github.com/microsoft/dotnet and generates an updated set of documentation.  @mairaw  I'm pretty sure this conflicts with #2674.  What do you want to do about that? Shall I wait for you to upgrade your includes.

## Suggested Reviewers
@mairaw

### CODE REVIEW

**Comment by mairaw** [comment]
I think it'd be better to wait that one to merge, otherwise it'll be a pain to resolve the merge conflicts. That one is just awaiting review. @rpetrusha can you review #2674 so we can get that one merged first?

No need to update the caps metadata. That's a metadata only relevant for our migration data.

**Comment by mairaw** [comment]
The PR was merged @conniey. Do you want to regenerate this?

**Comment by rpetrusha** [comment]
@conniey, we seem to have lost sight of your PR.  We've merged #2674. Can you fix the merge conflict, and then we'll review your PR? Sorry about the oversight.

**Comment by conniey** [comment]
@rpetrusha Hey Ron, I finally got the PortabilityService working with the latest changes and have updated this PR.  Hopefully when it finishes building. you can see the changes.

File: includes/migration-guide/retargeting/versionselector.md [hunk 1]
```diff
@@ -1,29 +1,37 @@
...
+> - [(4.0 | 4.6)](../../../../docs/framework/migration-guide/retargeting/4.0-4.6.md)
+> - [(4.0 | 4.6.1)](../../../../docs/framework/migration-guide/retargeting/4.0-4.6.1.md)
+> - [(4.0 | 4.6.2)](../../../../docs/framework/migration-guide/retargeting/4.0-4.6.2.md)
+> - [(4.0 | 4.7)](../../../../docs/framework/migration-guide/retargeting/4.0-4.7.md)
+> - [(4.0 | 4.7.1)](../../../../docs/framework/migration-guide/retargeting/4.0-4.7.1.md)
+> - [(4.5 | 4.5.1)](../../../../docs/framework/migration-guide/retargeting/4.5-4.5.1.md)
+> - [(4.5 | 4.5.2)](../../../../docs/framework/migration-guide/retargeting/4.5-4.5.2.md)
+> - [(4.5 | 4.6)](../../../../docs/framework/migration-guide/retargeting/4.5-4.6.md)
+> - [(4.5 | 4.6.1)](../../../../docs/framework/migration-guide/retargeting/4.5-4.6.1.md)
+> - [(4.5 | 4.6.2)](../../../../docs/framework/migration-guide/retargeting/4.5-4.6.2.md)
```
Comment (rpetrusha) [inline includes/migration-guide/retargeting/versionselector.md]: Connie, these links are all broken. The original versions (~/docs/framework/migration-guide... etc.) all work,

**Comment by rpetrusha** [comment]
@conniey Hey Connie, this looks good, except that a number of links to the retargeting app compat pages are broken (runtime all seem to be good). The links that begin ~/docs/framework all seem to work, but relative links are broken. If you can fix those and we can rebuild, we should be good to merge.

**Comment by conniey** [comment]
@rpetrusha It should be fixed now

**Review by rpetrusha — APPROVED** [review]
Thanks, @conniey. It is fixed, and the builds look good. I'll merge now.

**Pull request merged**